src/frontend/screens/ExportObservations.tsx

```diff
@@ -0,0 +1,210 @@
+    }
+    exportAndShare.mutate(
+      {exportType: typeToExport},
+      {
+        onError: err => {
+          Sentry.captureException(err);
+          navigation.navigate('ErrorBottomSheet');
+        },
+      },
+    );
+  }
+
+  return (
+    <BottomSheetWrapper>
+      <View style={{alignItems: 'center', gap: 10}}>
+        {observations.length > 0 && (
+          <>
+            <ExportOptionCard
+              title={formatMessage(m.allObservations)}
+              description={formatMessage(m.allObservationsDescription)}
+              isSelected={typeToExport === 'Observation'}
+              showError={showError && !typeToExport}
+              onPress={() => {
+                setTypeToExport('Observation');
+                if (showError) {
+                  setShowError(false);
+                }
+              }}
+            />
+            <ExportOptionCard
+              title={formatMessage(m.allObservationsAndMedia)}
+              description={formatMessage(m.allObservationsAndMediaDescription)}
+              isSelected={typeToExport === 'ObservationsWithMedia'}
+              showError={showError && !typeToExport}
+              onPress={() => {
+                setTypeToExport('ObservationsWithMedia');
+                if (showError) {
+                  setShowError(false);
+                }
+              }}
+            />
+          </>
+        )}
+        {tracks.length > 0 && (
+          <ExportOptionCard
+            title={formatMessage(m.tracks)}
+            description={formatMessage(m.tracksDescription)}
+            isSelected={typeToExport === 'Tracks'}
+            showError={showError && !typeToExport}
+            onPress={() => {
+              setTypeToExport('Tracks');
+              if (showError) {
+                setShowError(false);
+              }
+            }}
+          />
+        )}
+      </View>
+      {showError && (
+        <HeaderText
+          variant="header5"
+          style={{
+            color: WARNING_RED,
+            marginTop: 20,
+            textAlign: 'center',
+          }}>
+          {formatMessage(m.errorMessage)}
+        </HeaderText>
+      )}
+      {!exportAndShare.isPending ? (
+        <>
+          <PrimaryButton
+            fullSize={true}
+            onPress={handlePressDownload}
+            style={{marginTop: 20, alignSelf: 'center'}}
+            text={formatMessage(m.download)}
+          />
+          <SecondaryButton
+            fullSize={true}
+            onPress={() => navigation.goBack()}
+            style={{marginTop: 20, alignSelf: 'center'}}
+            text={formatMessage(m.close)}
+          />
+        </>
+      ) : (
+        <UIActivityIndicator style={{paddingTop: 60, paddingBottom: 40}} />
+      )}
+    </BottomSheetWrapper>
+  );
+};
+
+type ExportOptionCardProps = {
+  title: string;
+  description: string;
+  isSelected: boolean;
+  showError: boolean;
+  onPress: () => void;
+};
+
+export const ExportOptionCard = ({
```

Comment:
Text is cut off at the bottom for this component:

<img width="300" alt="image" src="https://github.com/user-attachments/assets/71bbd305-d03c-404d-8d17-5c85fed4315e" />

Reply:
weird, do you have any idea why this is happening?

Reply:
yeah it's because of the `flex: 1` you have on the `HeaderText` and `BodyText` components used in the component. If you remove that, you get the desired outcome:

<img width="300" alt="image" src="https://github.com/user-attachments/assets/adcae221-7e5e-45ac-b7ca-294d315dff88" />

Also, probably should add a right padding to account for text width changes (whether due to font size or translations, for example):

<img width="300" alt="image" src="https://github.com/user-attachments/assets/2484d6d6-d756-439f-a3c0-3a2956d0ff28" />

Reply:
> Also, probably should add a right padding to account for text width changes (whether due to font size or translations, for example):

Ah you already have the padding defined. The reason that the text isn't respecting it is due how text behaves in flex containers. Fix is to add `flex: 1` to the view that's wrapping the `HeaderText` and `BodyText`